samples/Java/Basic/EventProcessorSample/.classpath

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove these metadata files

add gitignore tags to repo's to make this easy (refer here: https://github.com/Azure/azure-event-hubs-java/blob/dev/.gitignore)

# eclipse java
.classpath
.project
*.prefs
/.metadata/